### ISSUE TITLE
cloc.sh out of date

### ISSUE THREAD

**Issue by muthutt** [tenstorrent/tt-metal#2406]
cloc.sh out of date

**Comment by TT-billteng** [comment]
@tt-rkim what do we use this script for?

**Comment by tt-rkim** [comment]
This is to count the lines of code in each component of the codebase.

Sr Leadership was very insistent on the optics of how many lines of code we have for specific components.

For example, some quotes are:

- "metal should have less than 10k lines of code"
- "models should be much less than 1k lines of code"

**Comment by TT-billteng** [comment]
do we still need this script?

**Comment by tt-rkim** [comment]
No, this was a push by leadership to constrain metal to as few lines of code as possible.

I'm okay nuking it and bringing it back if such an obsession returns.

**Comment by prajaramanTT** [comment]
@bkeith-TT Can we close this ticket ?

**Comment by tt-rkim** [comment]
yes
